delivery_readmes/DELIVERY.README.MethylSeq.txt

```diff
@@ -39,14 +39,14 @@ The 00-Report folder contains reports with sequencing information about the runs
 
 ── ProjectID/
    ├── 00-Reports/
-   │   ├── ProjectName_lanes_info.txt
-   │   ├── ProjectName_library_info.txt
-   │   ├── ProjectName_multiqc_report.html
-   │   ├── ProjectName_project_summary.html
-   │   ├── ProjectName_project_summary.md
-   │   └── ProjectName_sample_info.txt
-   └── manifestFiles/
-       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
+       ├── ProjectName_lanes_info.txt
+       ├── ProjectName_library_info.txt
+       ├── ProjectName_multiqc_report.html
+       ├── ProjectName_project_summary.html
+       ├── ProjectName_project_summary.md
+       ├── ProjectName_sample_info.txt
+       └── manifestFiles/
+            └── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
```

Comment:
Sorry, my bad,  "ProjectID" should be "SampleID" here actually.

Reply:
No problem, I didn't realise that. I'll fix it!